LambdaEngine/Include/Game/ECS/Systems/Networking/ClientSystem.h

```diff
@@ -49,7 +49,7 @@ namespace LambdaEngine
 		void Init();
 
 		void ReplayGameStatesBasedOnServerGameState(const GameState* pGameStates, uint32 count, const GameState& gameStateServer);
-		bool CompareGameStates(const GameState& gameStateLocal, const GameState& gameStateServer);
+		bool CompareGameStates(const GameState& gameStateServer, const GameState& gameStateLocal);
```

Comment:
Local first then server
```suggestion
		bool CompareGameStates(const GameState& gameStateLocal, const GameState& gameStateServer);
```